quirks/password-rules.json

```diff
@@ -104,6 +104,9 @@
     "bluesguitarunleashed.com": {
         "password-rules": "allowed: lower, upper, digit, [!$#@];"
     },
+    "bochk.com": {
+        "password-rules": "minlength: 8; maxlength: 12; required: lower; required: upper; required: digit; allowed: [#$%&()*+,.:;<=>?@_];"
```

Comment:
You can tack a `max-consecutive: 3;` onto this.

Reply:
Added `max-consecutive: 3;` to BOCHK rule